tests/readConcern/readconcern-getlevel-001.phpt

```diff
@@ -5,8 +5,6 @@ MongoDB\Driver\ReadConcern::getLevel()
 
 $tests = [
     null,
-    MongoDB\Driver\ReadConcern::LOCAL,
```

Comment:
Please either revert the changes to this file or keep at least one usage of a constant (e.g. `MongoDB\Driver\ReadConcern::LOCAL`). Related to my earlier comment, it's helpful for tests to demonstrate actual usage of the classes.

In this case, retaining a single constant here would cover all of the bases for construction:

 * no level
 * a known, supported level (either a literal corresponding to a constant or the constant itself)
 * a known, unsupported level

Reply:
done